src/index.js

```diff
@@ -43,6 +45,7 @@ const VueHtmlToPaper = {
         if (localOptions.specs) specs = localOptions.specs;
         if (localOptions.replace) replace = localOptions.replace;
         if (localOptions.styles) styles = localOptions.styles;
+        if (localOptions.autoClose) autoClose = localOptions.autoClose;
```

Comment:
The problem with this approach (and the approach in the original code too) is that if you pass a _falsy_ value as an option (autoClose = false), it will never enter in this if statement... so if you want to leave it open, this won't work 🤔